plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/lsp/AmazonQLspService.kt

```diff
@@ -318,7 +318,7 @@ private class AmazonQServerInstance(private val project: Project, private val cs
 
         DefaultAuthCredentialsService(project, encryptionManager, this)
         TextDocumentServiceHandler(project, this)
-        WorkspaceServiceHandler(project, this)
+        WorkspaceServiceHandler(project, initializeResult, this)
```

Comment:
if you rebase, you should have access to the server initialize result directly instead of needing to resolve the deferred value